CRM/Core/Permission.php

```diff
@@ -1137,6 +1137,9 @@ public static function getEntityActionPermissions() {
     $permissions['line_item'] = $permissions['contribution'];
 
     $permissions['financial_item'] = $permissions['contribution'];
+    $permissions['financial_type']['get'] = $permissions['contribution']['get'];
```

Comment:
Initial test runs suggest that maybe we want the permissions of contribution.get OR 'Administer CiviCRM' (or 'Administer CiviCRM Data' which is implicit in Administer CiviCRM) 

 - since I guess lines like this suggest that we need to be careful not to reduce access https://github.com/civicrm/civicrm-core/blob/0fba5cf89d707bc8f305bc9f12a3832c110a11c7/CRM/Event/BAO/Event.php#L2364-L2366

I just added error handling - but not 100% sure if that is enough forgiveness